lms/models/_mixins.py

```diff
@@ -0,0 +1,19 @@
+import datetime
+
+import sqlalchemy as sa
+
+
+class TimestampedModelMixin:
+    created = sa.Column(
+        sa.DateTime(),
+        server_default=sa.func.now(),
+        default=datetime.datetime.utcnow,
+        nullable=False,
+    )
+    updated = sa.Column(
+        sa.DateTime(),
+        default=datetime.datetime.utcnow,
+        server_default=sa.func.now(),
+        onupdate=sa.func.now(),
```

Comment:
Only change over the existing one is onupdate using func.now() vs datetime

Reply:
There's some prior art for this in `h`, but I don't think we should take that too seriously: https://github.com/hypothesis/h/blob/master/h/db/mixins.py

One thing to note is the mixed use of Python / Postgres functions for specifying the date. From what I can see (https://docs.sqlalchemy.org/en/14/core/defaults.html#client-invoked-sql-expressions) you can use the server side version for the `default` and `onupdate`. Without knowing the exact outcome, it seems better to have an identical mechanism for setting them all if we can.

This insulates us from any drift between Postgres and pPthon